ui/pages/add-collectible/add-collectible.js

```diff
@@ -65,6 +67,17 @@ export default function AddCollectible() {
       );
     }
     dispatch(setNewCollectibleAddedMessage('success'));
+
+    trackEvent({
+      event: 'Collectible Added',
+      category: 'Wallet',
+      sensitiveProperties: {
+        token_contract_address: address,
+        tokenId: tokenId.toString(),
+        source: 'manual',
```

Comment:
we're missing properties here. asset_type, token_standard... and source should 'custom' (if this is when the user manually types in all the details of the token)

Reply:
fixed [here](https://github.com/MetaMask/metamask-extension/pull/14279/commits/a82b0a8b6ff401b1be075480a26a729d3ba46c24)